docs/Plugins.md

```diff
@@ -0,0 +1,28 @@
+Plugins
+=======
+
+Moya plugins are used to perform some side effect whenever a request is sent or received. Plugins are defined by the `PluginType` protocol, which can receive callbacks when a request is going to be sent, and when a response has been received. 
+
+##Built in plugins##
+Moya ships with some default plugins which can be used for common functions: authentication, network activity indicator management, and logging.
+
+####Authentication####
```

Comment:
Might make sense to remove the trailing `###` here, to be consistent with the other docs?